Makefile

```diff
@@ -6,11 +6,23 @@ BINDIR ?= output
 USE_VENDORIZED_BUILD_HARNESS ?=
 
 ifndef USE_VENDORIZED_BUILD_HARNESS
--include $(shell curl -s -H 'Authorization: token ${GITHUB_TOKEN}' -H 'Accept: application/vnd.github.v4.raw' -L https://api.github.com/repos/open-cluster-management/build-harness-extensions/contents/templates/Makefile.build-harness-bootstrap -o .build-harness-bootstrap; echo .build-harness-bootstrap)
+-include $(shell curl -s -H 'Authorization: token ${GITHUB_TOKEN}' -H 'Accept: application/vnd.github.v4.raw' -L https://api.github.com/repos/stolostron/build-harness-extensions/contents/templates/Makefile.build-harness-bootstrap -o .build-harness-bootstrap; echo .build-harness-bootstrap)
 else
 -include vbh/.build-harness-vendorized
 endif
 
+.PHONY: deps fmt lint test coverage build build-images
+
+# GITHUB_USER containing '@' char must be escaped with '%40'
+GITHUB_USER:=$(shell echo $(GITHUB_USER) | sed 's/@/%40/g')
+GITHUB_TOKEN?=
+
+GOPRIVATE=github.com/stolostron
+
+ifdef GITHUB_TOKEN
+$(shell git config --global url."https://$(GITHUB_TOKEN):x-oauth-basic@github.com/".insteadOf "https://github.com/")
+endif
```

Comment:
I'm curious why none of the other PRs have this change.

Reply:
After 2.2, the endpoint-operator repo was moved to klusterlet-addon and is public. But, that change is not ported over to 2.2. endpoint-operator is an indirect dependency for search-collector but is a private repo. So, to pull it had to add this similar to app team's PR https://github.com/stolostron/multicloud-operators-placementrule/tree/release-2.2.

Reply:
Thanks! we had a similar line in the .travis file.